### PR TITLE
fixed CNCF Landscape iframe uses wrong colors

### LINKED ISSUE #44803: CNCF Landscape iframe uses wrong colors
**This is a Bug Report**

**Problem:**
https://github.com/kubernetes/website/blob/06fb191b99a1a248cc8c697cd5170aea195412cd/layouts/shortcodes/cncf-landscape.html uses the wrong background color(s); it's `#d95e00` but should be `#3371e3` (Kubernetes blue). I think the orange color was copied from https://github.com/kubernetes/website/blob/06fb191b99a1a248cc8c697cd5170aea195412cd/data/announcements/scheduled.yaml#L214 by mistake.

**Proposed Solution:**
Revise the shortcode. Consider making the color be set once and then reused (“[DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself)”).

**Page to Update:**
You would see the change on https://kubernetes.io/partners/

We should use blue backgrounds whether or not an announcement banner is showing.

**Additional Information:**
PR https://github.com/kubernetes/website/pull/44792 updated the shortcode.

/area web-development
/triage accepted
/priority backlog

---

If fixing this, consider also moving the iframe scroll-margin-top fix to be more general.
See https://github.com/kubernetes/website/pull/44792#discussion_r1458153262 and https://github.com/kubernetes/website/issues/29909

### PR DESCRIPTION
**Pull request by Manish-hr14**
fixed #44803

### CODE REVIEW

File: data/announcements/scheduled.yaml [hunk 1]
```diff
@@ -211,7 +211,7 @@ announcements:
 - name: Legacy package repositories shutdown
   startTime: 2023-11-27T00:00:00
   endTime: 2024-02-01T00:00:00
-  style: "background: #d95e00"
+  style: "background: #3371e3"
```
Comment (dipesh-rawat) [inline data/announcements/scheduled.yaml:214]: This proposed change is incorrect. It alters the color of the announcement banner (click [preview link](https://deploy-preview-44808--kubernetes-io-main-staging.netlify.app/)) instead of the CNFC iframe as requested.

File: layouts/blog/baseof.html [hunk 2]
```diff
@@ -6,6 +6,7 @@
     {{ end }}
     {{ partial "head.html" . }}
     <title>{{ if .IsHome }}{{ .Site.Title }}{{ else }}{{ with .Title }}{{ . }} | {{ end }}{{ .Site.Title }}{{ end }}</title>
+
```
Comment (dipesh-rawat) [inline layouts/blog/baseof.html:9]: Why this change?

**Comment by sftim** [comment]
/hold
https://github.com/kubernetes/website/pull/44808#discussion_r1458947462

Let's not merge this incorrect change by accident. OK to unhold once a reviewer confirms the fix is along the right lines.
